examples/pytorch_surrogate_model/run_ml_surrogate.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Ryan Sandberg, Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from surrogate_model_definitions import surrogate_model
+import torch
```

Comment:
For the purpose of testing, let's do a `try` block here and exit with zero-error but a warning that the test was skipped (in case torch cannot be found).

Rationale: does not introduce a mandatory `torch` dependency on ImpactX to pass its tests.

Reply:
Let's install `pytorch` via pip to one of our CI tests, e.g., one of the Ubuntu tests.